app/templates/fx_ios/release/listing_sept_2015.php

```diff
@@ -117,6 +117,9 @@
 
 $whatsnew = function ($translations) use ($_) {
     return <<<OUT
-• {$_('Tracking Protection is now on by default for regular and private browsing! We’ve also improved the iPad experience with keyboard shortcuts and better tab navigation.')}
+{$_('This update contains stability improvements and bug fixes.')}
```

Comment:
This should be: Our latest release includes features that can help you be more productive.